buildsystem/cpp.cmake

```diff
@@ -88,7 +89,7 @@ function(finalize_binary target_name output_name type)
 				message(FATAL_ERROR "finalize_binary flag 'allow_no_undefined' is only valid for libraries!")
 			endif()
 
-			if(NOT "${CMAKE_CXX_FLAGS}" MATCHES "-fsanitize")
+			if(NOT "${CMAKE_CXX_FLAGS}" MATCHES "-fsanitize" AND NOT MSVC)
```

Comment:
openage is and has to be a dynamic library so python can load it (`import libopenage`). This can never be done for a static library.

On Linux it did not complain, which is why we added those options. Hmm.